### PR TITLE
Prototype implementation

### PR DESCRIPTION
**Pull request by tomlovesgithub**
changes to 
- Navbar
- Diversity Page
- Header 
- Favourites button

### CODE REVIEW

File: src/components/Page/AddToFavouritesButton.vue [hunk 1]
```diff
@@ -1,20 +1,9 @@
 <template>
   <button
     ref="updateFavoutitesBtn"
-    class="govuk-button govuk-button--secondary govuk-!-margin-0"
+    class="govuk-button govuk-button--secondary govuk-!-margin-top-4 float-right"
```
Comment (warrensearle) [inline src/components/Page/AddToFavouritesButton.vue:4]: Don't think we should have layout styles like `float-right` within the component itself. Please review.

File: src/App.vue [hunk 2]
```diff
@@ -1,77 +1,72 @@
 <template>
   <div class="govuk-width-container">
-    <div
-      id="header"
```
Comment (warrensearle) [inline src/App.vue:4]: header id is used in the print stylesheet, so is important

File: src/App.vue [hunk 3]
```diff
@@ -92,13 +87,44 @@ export default {
...
+  background-color: #fafafa;
+  padding: 1.25em 0.625em 0 0.625em;
+};
+.govuk-main-wrapper {
+  padding-top: 0px;
+}
+  .govuk-grid-column-one-half {
+    width: 100%;
+    float: left;
+}
```
Comment (lloback) [inline src/App.vue:102]: The tabulation is not correct on this file. All along.

File: src/styles/main.scss [hunk 4]
```diff
@@ -70,6 +83,10 @@ body {
   color: #0b0c0c;
 }
 
+.govuk-width-container {
+  max-width: 1200px;
```
Comment (lloback) [inline src/styles/main.scss:87]: can we use some variables and not add random values?
(THis is question, not a comment)

File: src/views/Exercises/Show.vue [hunk 5]
```diff
@@ -128,13 +129,13 @@ export default {
             { page: 'Applied', name: 'exercise-show-applications-in-status', params: { status: 'applied' } },
           ],
         });
-        // pages.push({ page: 'Independent assessments', name: 'exercise-show-independent-assessments' });
+        pages[1].children.push({ page: 'Independent Assessments', name: 'exercise-show-independent-assessments' });
```
Comment (lloback) [inline src/views/Exercises/Show.vue:132]: this line is conflicting

File: src/views/Exercises/Show.vue [hunk 6]
```diff
@@ -126,15 +127,15 @@ export default {
           children: [
             { page: 'Draft', name: 'exercise-show-applications-in-status', params: { status: 'draft' } },
             { page: 'Applied', name: 'exercise-show-applications-in-status', params: { status: 'applied' } },
+            { page: 'Independent Assessments', name: 'exercise-show-independent-assessments' },
```
Comment (lloback) [inline src/views/Exercises/Show.vue:130]: @tomlovesgithub thank you for doing the change here - it makes more sense and you don't rely on the Array Index.

Comment (tomlovesgithub) [reply src/views/Exercises/Show.vue:130]: Thanks for pointing it out, completely missed me!

**Pull request merged**